app/forms/collection_form.rb

```diff
@@ -14,7 +14,7 @@ class CollectionForm < Reform::Form
   property :depositor_sunets, virtual: true, prepopulator: lambda { |_options|
     self.depositor_sunets = depositor_sunets_from_model.join(', ')
   }
-  property :review_enabled, virtual: true, default: -> { model.reviewers.present? ? 'true' : 'false' }
+  property :review_enabled, virtual: true, default: -> { model.review_enabled? ? 'true' : 'false' }
```

Comment:
and this change is also part of new functionality?

Reply:
I created `#review_enabled?` to be used by new functionality, so I'm also using it here to avoid duplicate code.